ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -65,6 +65,9 @@ public CCDCase to(Claim claim) {
             .submitterId(claim.getSubmitterId())
             .submitterEmail(claim.getSubmitterEmail())
             .issuedOn(claim.getIssuedOn())
+            .state(claim.getState()
```

Comment:
As `state` is driven by CCD events, and stored as property of CaseDetails in CCD. Duplicating this in case data will cause more confusion in my opinion. We are already  mapping this back to `Claim.state` in `from` mapping of CCD. We don't need `to` mapper for this.

Reply:
👍